src/Transforms/Image.php

```diff
@@ -0,0 +1,64 @@
+<?php namespace Leean\Acf\Transforms;
+
+/**
+ * Class Image.
+ *
+ * @package Leean\Acf\Transforms
+ */
+class Image
+{
+	/**
+	 * Apply the transforms
+	 *
+	 * @param array $field The field.
+	 * @return array
+	 */
+	public static function apply( $field ) {
+		if ( ! ( 'image' === $field['type'] && 'id' === $field['return_format']) ) {
+			return $field['value'];
+		}
+
+		self::transform_image_fields( $field );
+
+		return $field['value'];
+	}
+
+	/**
+	 * Do the image size transform.
+	 *
+	 * @param string $field			Field
+	 * @return array
+	 */
+	public static function transform_image_fields( &$field ) {
```

Comment:
My idea here is that it makes it easier to chain the transforms (eg in Repeater we have 2). The transform update the field['value'] param, then it's passed to the next one to do the same. I think it gets a bit more confusing if we return a value then have to pass it to the next transform - for one thing we would have the $value and $field['value'] both with different values.

But I'm open to changing this if you think there's a better way?